src/controller/user.controller.ts

```diff
@@ -0,0 +1,123 @@
+import { Request, Response } from "express";
+import { NOT_FOUND, INTERNAL_SERVER_ERROR, OK, CREATED ,SocialMediaType, BAD_REQUEST } from "../utils/constants.utils";
+import userService from "../services/user.service";
+import { ISocialMediaURLS, IUser } from "../models/interfaces.models";
+import { ICreatorWorkMetadata } from "../models/interfaces.models";
+import { forEachChild } from "typescript";
+
+
+class UserController {
+
+    // get all the users
+    getCreatedUser = async (request: Request, response: Response) => {
+        try {
+            const users = await userService.getUser();
+            if (users) {
+                return response.status(OK).send(users);
```

Comment:
Our standard is to always have a message property in the response we are sending from any controller method

Reply:
this should be 
```
if (!user || users.length < 1) {return a 404 response saying no user found}

return response.status(OK).send({data: users, message: "Users retrieved Successfully"})
```